test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -498,6 +509,7 @@
       measurePods: 2000
   - name: 5000Nodes_5000Pods
     labels: [performance]
+    threshold: 85
```

Comment:
I looked at the performance of this a few times in the past... not sure if we can do much more, but it might be worth looking at it.